types/bprpc.go

```diff
@@ -181,3 +185,14 @@ type QuerySQLChainProfileResp struct {
 	proto.Envelope
 	Profile SQLChainProfile
 }
+
+type QueryTxStateReq struct {
+	proto.Envelope
+	Hash hash.Hash
+}
+
+type QueryTxStateResp struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type QueryTxStateResp should have comment or be unexported